src/DataBus/ConfigureAzureDataBus.cs

```diff
@@ -162,31 +145,14 @@ public static DataBusExtensions<AzureDataBus> CleanupInterval(this DataBusExtens
         }
 
         /// <summary>
-        /// Sets token credential to authenticate with Storage Blob service
-        /// <remarks>Token credentials can be created using <see cref="AzureServiceTokenProvider"/> with token renewal configured.</remarks>
+        /// For advanced authentication scenarios, provide a properly configured BlobContainerClient instead of a connectionstring
         /// </summary>
-        /// <param name="config">The configuration to modify.</param>
-        /// <param name="storageAccountName">The storage account name.</param>
-        /// <param name="renewalTimeBeforeTokenExpires">The renewal time before token expiry.</param>
-        /// <param name="endpointSuffix">Endpoint suffix associated with storage blob service per type of data center. Default is "core.windows.net" for public Azure data centers.</param>
-        public static DataBusExtensions<AzureDataBus> AuthenticateWithManagedIdentity(this DataBusExtensions<AzureDataBus> config, string storageAccountName, TimeSpan renewalTimeBeforeTokenExpires, string endpointSuffix = "core.windows.net")
+        public static DataBusExtensions<AzureDataBus> BlobContainerClient(this DataBusExtensions<AzureDataBus> config,
```

Comment:
Should this configuration API called `.UseBlobContainerClient()` rather than `.BlobContainerClient()`?